components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/java/org/wso2/carbon/apimgt/rest/api/store/utils/mappings/APIMappingUtil.java

```diff
@@ -210,6 +210,27 @@ public static APIListDTO fromAPIListToDTO(List<API> apiList, int offset, int lim
         return apiListDTO;
     }
 
+    /**
+     * Converts a List object of APIs into a DTO
+     *
+     * @param apiList List of APIs
+     * @return APIListDTO object containing APIDTOs
+     */
+    public static APIListDTO fromAPIListToDTO(List<API> apiList) {
+        APIListDTO apiListDTO = new APIListDTO();
+        List<APIInfoDTO> apiInfoDTOs = apiListDTO.getList();
+        if (apiInfoDTOs == null) {
```

Comment:
The APIListDTO class creates a new APIInfoDTO list when you are creating a new object out of APIListDTO class. Hence we can remove this null check.